apps/src/code-studio/components/DownloadReplayVideoButton.jsx

```diff
@@ -239,7 +239,10 @@ class DownloadReplayVideoButton extends React.Component {
   };
 
   shouldRenderButton() {
-    return this.props.channelId && this.hasReplayVideo();
+    // this is temporarily disabled until we decide for sure that we want to remove
```

Comment:
There seems to be a lot of white space on the righthand side of dialog now. But I see that's consistent with other labs like Sprite lab projects. 
Could you confirm that this button is only used in Dance Party projects?

Reply:
Dance Party is the only place I've managed to see it show up.

Reply:
> Could you confirm that this button is only used in Dance Party projects?

It looks like [this](https://github.com/code-dot-org/code-dot-org/blob/b1b8a5b246b21706a56e6fdf6669de242ec5d188/apps/src/code-studio/components/DownloadReplayVideoButton.jsx#L125) line guarantees that this button only shows for Dance Party.